src/main/java/com/example/petbuddybackend/entity/amenity/AnimalAmenity.java

```diff
@@ -0,0 +1,26 @@
+package com.example.petbuddybackend.entity.amenity;
+
+import com.example.petbuddybackend.entity.animal.Animal;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Getter @Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+@EqualsAndHashCode(of = "id")
+public class AnimalAmenity {
```

Comment:
Tu nie lepszy byłby klucz złożony z kluczy obcych Amenity i Animal? Jeśli się mylę to mnie popraw, ale chyba nie chcemy duplikatów to samo amenity z tym samym animal?

Reply:
tak, sztuczny klucz jest dodany tylko, zeby bylo latwiej potem tą klase gdzies podpinac i on tez bedzie unikalny